regression/regression_utils.py

```diff
@@ -31,6 +32,38 @@
 """
 
 
+class RetryTestsMetaclass(type):
+
+    NUM_RETRIES = 2
+    FLAKY_ERROR_CLASSES = (AssertionError,)
```

Comment:
The issue you pointed to was a 404, not an assertion failure.  Again, I'm not sure what kind of "flakiness" would result in an assertion that failed one time, but passed the next:  ISTM that network failures should raise errors *before* we get around to making assertions about the results.